openfe/setup/ligand_network_planning.py

```diff
@@ -84,6 +85,8 @@ def generate_radial_network(
     edges = []
 
     for ligand in ligands:
+        if ligand == central_ligand:
```

Comment:
No warning? Or is this something that a user would expect we handle in this way? I think it makes sense to skip it this way the more I think about it

Reply:
Probably would second that.